linters/prisma/plugin.yaml

```diff
@@ -0,0 +1,15 @@
+version: 0.1
+lint:
+  definitions:
+    - name: prisma
+      tools: [prisma]
+      files: [prisma]
+      known_good_version: 4.16.1
```

Comment:
We'll want to add in `suggest_if: never` (maybe eventually "if_files") for explicitness

Reply:
Done